packages/mp/src/mp/core/validators.py

```diff
@@ -168,3 +168,22 @@ def validate_png_content(path: pathlib.Path) -> bytes:
     except UnidentifiedImageError as e:
         msg = f"File is not a valid image or is corrupted: {path}"
         raise ValueError(msg) from e
+
+
+def coerce_bool_from_str_or_none(v: Any) -> bool:  # noqa:ANN401
```

Comment:
Since it's from str or none:

```suggestion
def coerce_bool_from_str_or_none(v: str | None) -> bool:
```

That way it will notify someone if they are passing a parameter that should only be an int for example